packages-ts/terra-sdk/src/index.ts

```diff
@@ -0,0 +1,55 @@
+import { WebSocketClient } from '@terra-money/terra.js';
+import { TxLog, Int } from '@terra-money/terra.js';
+
+export class OCR2Feed {
+    private _wsClient: WebSocketClient;
+
+    constructor(readonly client: WebSocketClient){
+        this._wsClient = client;
+    }
+
+    public start() {
+        this._wsClient.start()
+    }
+
+    public destroy() {
+        this._wsClient.destroy()
+    }
```

Comment:
If `_wsClient` is passed in as a param when making this class, shouldn't we expect that the caller takes care of start/destroy?

Reply:
Ah yes - just switched the constructor from url, so this can be simpler now.

Reply:
Yeah, the suggestion makes sense and the API needs more thought as we didn't have the time currently.

That being said, it should be ok as these are public methods currently (`start/destroy`) and not called internally. So it's up to the user to use them if desired.